src/native/minipal/thread.h

```diff
@@ -71,12 +67,14 @@ static inline size_t minipal_get_current_thread_id(void)
         tid = (size_t)find_thread(NULL);
 #elif defined(__sun)
         tid = (size_t)pthread_self();
-#else
+#elif defined(__wasm) && !defined(_REENTRANT)
```

Comment:
```suggestion
#elif defined(__wasm)
```
I expect we would get a build break for multithreaded wasm otherwise. (Multi-threaded wasm is an optional CI leg.)